graph/patterns/enums.md

```diff
@@ -1,25 +1,33 @@
-### Enums
+# Enums
 
-In OData, enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
+Microsoft Graph API Design Pattern
 
-```xml
-<EnumType Name="color">
-    <Member Name="Red" Value="0" />
-    <Member Name="Green" Value="1" />
-    <Member Name="Blue" Value="2" />
-</EnumType>
-```
+Enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
+
+## Problem
+
+The API Design requires that a specific property of a data structure has a finite and limited set of values, and such limitation needs to be clearly communicated to the user and give the developer the same structure
+
+## Solution
+
+In OData, Enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
 

```

Comment:
Enum solution must be based on the evolvable enum. Not evolvable enums are exceptions.